mappings/net/minecraft/container/CombinationContainer.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_4861 net/minecraft/container/CombinationContainer
+	FIELD field_22479 resultInv Lnet/minecraft/class_1263;
+	FIELD field_22480 inputsInv Lnet/minecraft/class_1263;
```

Comment:
In `CraftingTableContainer`, the names of the inventory fields are `craftingInv` and `resultInv`. Perhaps those should be changed as well.

Reply:
hmm, I suggest the name because there is no input/result slots.

Reply:
> Perhaps those should be changed as well.
They should.

Reply:
@Runemoro If we do change it, it would have to be in a different PR, perhaps with the `updateResult` changes as well.

Reply:
Yes, but why give these fields a bad name just because other fields have a bad name?

Reply:
It’s consistency over correctness. They can always be consistent and correct later.

Reply:
So if the other one had a typo, like `reusltsInv`, you'd copy the typo here too?

Reply:
@Runemoro That’s a spelling error rather than a break in convention. In my opinion, whether to use the existing mappings for another mapping depends on the situation; there is a difference between a typo, an inconsistency, and an innaccuracy.

Reply:
Shortening "inventory" to three-letters isn't a convention. It breaks the convention of not using uncommon abbreviations and is just as bad as a spelling mistake. Also, there are hundreds of other names that use the word "inventory" rather than "inv".

Reply:
@Runemoro It breaks the written convention of not using uncommon abbreviations, but the actual convention (at least within the containers‘ mappings) is to suffix it with `inv` when there are multiple inventories within the same container.

Reply:
Issue #1153 has been opened to track the removal of the `inv` suffix.